modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -38,10 +38,27 @@ If you disable the Inbound Traffic tab, changes can take several minutes to comp
 . In the *Resource Allocation* section:
 +
 The inbound load balancer runs on each controller node configured in Runtime Fabric. Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
+
+.. Select a mode.
+
+*** *Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified.
+
+*** *Dedicated Mode* is only enabled if one or more dedicated internal load balancer nodes are available in Runtime Fabric. Because dedicated internal load balancer nodes use all available resources for running the internal load balancer, you cannot specify the amount of CPU cores and memory. If you are setting up a Runtime Fabric with dedicated nodes, include the IP addresses of the dedicated load balancer nodes only when configuring the external load balancer.
```

Comment:
ditto
```suggestion
* *Dedicated Mode* is enabled only if one or more dedicated internal load balancer nodes is available in Runtime Fabric. 
Because dedicated internal load balancer nodes use all available resources for running the internal load balancer, you cannot specify the number of CPU cores and amount memory. If you are setting up a Runtime Fabric instance with dedicated nodes, include the IP addresses of the dedicated load balancer nodes only when configuring the external load balancer.
```